lib/src/application/auth_bloc/auth_state.dart

```diff
@@ -4,7 +4,7 @@ part of 'auth_bloc.dart';
 class AuthState with _$AuthState {
   factory AuthState({
     required bool status,
-    required bool isCheckAuth,
+    required Status isCheckAuth,
```

Comment:
change the name of variable `isCheckAuth` to more readable and relatable to status like `checkAuthStatus`